govcd/sample_govcd_test_config.yaml

```diff
@@ -132,6 +132,9 @@ vcd:
     # A vSphere server name for creating an external network
     vimServer: vc9
     #
+    # IP of a pre-configured LDAP server
+    # using Docker image https://github.com/rroemhild/docker-test-openldap
+    ldap_server: 10.10.10.99
```

Comment:
Just like in the test struct, could you also remove the `misc.ldapContainer` documentation below

Reply:
done